pkg/harvester/models/devices.harvesterhci.io.migconfiguration.js

```diff
@@ -0,0 +1,98 @@
+import SteveModel from '@shell/plugins/steve/steve-class';
+import { escapeHtml } from '@shell/utils/string';
+import { colorForState } from '@shell/plugins/dashboard-store/resource-class';
+
+/**
+ * Class representing vGPU MIGConfiguration resource.
+ * @extends SteveModal
+ */
+export default class MIGCONFIGURATION extends SteveModel {
+  get _availableActions() {
+    let out = super._availableActions;
+
+    out = out.map((action) => {
+      if (action.action === 'goToEditYaml') {
+        return { ...action, enabled: true };
```

Comment:
Should we disable `Edit YAML` when MIG is enabled? similar to how `Edit Config` behaves.